amazonlinux2.dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM amazonlinux:2.0.20200722.0
```

Comment:
It's annoying that, unlike for Amazon Linux 1, they don't publish the precise AMI.
https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

(I guess it's not a huge deal, but worth noting.)